src/preprocessing/passes/non_clausal_simp.cpp

```diff
@@ -69,7 +69,7 @@ NonClausalSimp::NonClausalSimp(PreprocessingPassContext* preprocContext)
 PreprocessingPassResult NonClausalSimp::applyInternal(
     AssertionPipeline* assertionsToPreprocess)
 {
-  Assert(options::unsatCoresMode() != options::UnsatCoresMode::OLD_PROOF
+  Assert(options::unsatCoresMode() == options::UnsatCoresMode::OFF
```

Comment:
I think this assertion should just be deleted now.

Reply:
Indeed. Done.